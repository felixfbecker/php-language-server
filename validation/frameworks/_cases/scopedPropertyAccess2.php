<?php

namespace MyNamespace;

$process = new A;
array($process::OUT, '123');