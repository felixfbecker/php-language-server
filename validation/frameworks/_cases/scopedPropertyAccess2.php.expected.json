{
    "references": {
        "MyNamespace\\A": [
            "./_cases/scopedPropertyAccess2.php"
        ]
    },
    "definitions": {
        "MyNamespace": {
            "extends": [],
            "isGlobal": true,
            "isStatic": false,
            "canBeInstantiated": false,
            "symbolInformation": {
                "name": "MyNamespace",
                "kind": 3,
                "location": {
                    "uri": "./_cases/scopedPropertyAccess2.php"
                },
                "containerName": ""
            },
            "type__class": "LanguageServer\\Tests\\ValidationTest",
            "type": null,
            "documentation": null
        }
    }
}